image/image.go

```diff
@@ -117,6 +117,11 @@ func Prepare(opts *Options) error {
 		return err
 	}
 
+	// TODO: might make sense to support this later
+	if model.Classic() {
+		return fmt.Errorf("classic models are not supported")
```

Comment:
Perhaps giving a hint of what is unsupported:

`"cannot prepare image of a classic model"`